core/frame/meta_frame.go

```diff
@@ -1,43 +1,62 @@
 package frame
 
 import (
-	"strconv"
 	"time"
 
 	"github.com/yomorun/y3"
 )
 
-// MetaFrame is a Y3 encoded bytes, SeqID is a fixed value of TYPE_ID_TRANSACTION.
-// used for describes metadata for a DataFrame.
+const (
+	// LoadBalanceRandomPick means Zipper will choose SFN instance by a load balance strategy.
+	LoadBalanceRandomPick byte = 0x00
+	// LoadBalanceBindInstance means data will be sent to the specific SFN according to instanceID.
+	LoadBalanceBindInstance byte = 0x01
+	// LoadBalanceBroadcast means data will be sent to all SFN instances.
+	LoadBalanceBroadcast byte = 0x02
```

Comment:
which scenario will be used in for this algorithm?

Reply:
https://github.com/xileteam/yomo-use-case-socks5/pull/9/files#diff-746b2b7fb72c2ec38c6ded7abe807bdf4ed6b1c08f09851efbb406c3691a4175R47

Reply:
emm, why zipper need boradcast request to all the instances?

Reply:
to mimic sticky session...